### PR TITLE
[MM-58200] Log joining session remote address

### PR DESCRIPTION
**Pull request by streamer45**
#### Summary

Logging the information added in https://github.com/mattermost/mattermost/pull/27332

@esarafianou This PR makes us log a user's IP address when joining a call, as requested by a customer, as there's currently no other way to audit clients joining calls. Let me know if there are concerns.

#### Ticket Link

https://mattermost.atlassian.net/browse/MM-58200

### CODE REVIEW

File: server/websocket.go [hunk 1]
```diff
@@ -744,7 +750,10 @@ func (p *Plugin) handleJoin(userID, connID, authSessionID string, joinData Calls
 			})
 		}
 
-		p.LogDebug("session has joined call", "userID", userID, "sessionID", connID, "channelID", channelID, "callID", state.Call.ID)
+		p.LogDebug("session has joined call",
```
Comment (esarafianou) [inline server/websocket.go:753]: I understand that this is logged only in the log level in the server is set to Debug, right?

Comment (streamer45) [reply server/websocket.go:753]: Yep

**Review by esarafianou — APPROVED** [review]
No issues from the security/compliance side.

**Review by cpoile — APPROVED** [review]
Thanks!

**Pull request merged**